dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/SingularityIT.java

```diff
@@ -28,7 +27,10 @@
     @ClassRule
     public static TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private static final String CONFIG = ResourceHelpers.resourceFilePath("config_for_singularity");
```

Comment:
I would name the variable SINGULARITY_CONFIG or SINGULARITY_CONFIG_TEMPLATE or something like that; avoids any confusion further down when it is used.